tests/console/yast2_ntpclient.pm

```diff
@@ -69,72 +97,100 @@ sub run {
+        send_key_until_needlematch 'yast2_ntp-client_sync_server', 'alt-s';
+        send_key 'alt-n';
+
+        # check ntp server is displayed for changes
+        assert_screen 'yast2_ntp-client_ntp_server';
+
+        # if support-server is used we need to select it
+        if (check_var('USE_SUPPORT_SERVER', 1)) {
+            type_string "$ntp_server";
+        }
+        else {
+            # select public ntp server
+            send_key 'alt-s';
+            assert_screen 'yast2_ntp-client_ntp_server_public';
+            wait_screen_change { send_key 'down'; };
+            send_key 'ret';
+
+            # check public ntp server is showing up and select UK
+            assert_screen 'yast2_ntp-client_public_ntp_server_opened';
+            send_key 'alt-u';
+            assert_screen 'yast2_ntp-client_public_ntp_country';
+            send_key_until_needlematch 'yast2_ntp-client_country_uk', 'up';
+            wait_screen_change { send_key 'ret'; };
+            send_key 'alt-s';
+            assert_screen 'yast2_ntp-client_public_ntp_server_uk';
+            send_key 'ret';
+        }
+    }
 
     # run test
     send_key 'alt-t';
-    assert_screen 'yast2_ntp-client_public_ntp_test';
+    assert_screen ['bsc#1074726', 'yast2_ntp-client_public_ntp_test'];
+
+    # If NTP server test failed, it's certainly because bsc#1074726 bug
+    record_soft_failure 'bsc#1074726' if (match_has_tag 'bsc#1074726');
 
     # close it with OK
-    send_key_until_needlematch "yast2_ntp-client_public_ntp_server_added", "alt-o", 3, 5;
-
-    # now check display log and save log
-    send_key 'alt-l';
-
-    # check log got displayed and go to Advanced
-    assert_screen 'yast2_ntp-client_display_log';
-    send_key 'alt-v';
-    assert_screen 'yast2_ntp-client_display_log_save_log';
-    send_key 'ret';
-
-    # give a new file name
-    assert_screen 'yast2_ntp-client_save_log_as';
-    send_key 'alt-f';
-    type_string 'ntpclient.log';
-    assert_screen 'yast2_ntp-client_new_file_name';
-    send_key 'alt-o';
-    assert_screen 'yast2_ntp-client_display_log';
-    wait_still_screen 1;
-    send_key 'alt-c';
-    # Assert that we are back to configuration page
-    assert_screen 'yast2_ntp-client_configuration';
-    wait_still_screen 1;
+    my $ntp_client_needle = check_var('USE_SUPPORT_SERVER', 1) ? 'support_server' : 'public';
+    send_key_until_needlematch "yast2_ntp-client_${ntp_client_needle}_ntp_server_added", "alt-o", 3, 5;
+
+    if (!$sle_or_leap_at_least_15) {
+        # now check display log and save log
+        send_key 'alt-l';
+
+        # check log got displayed and go to Advanced
+        assert_screen 'yast2_ntp-client_display_log';
+        send_key 'alt-v';
+        assert_screen 'yast2_ntp-client_display_log_save_log';
+        send_key 'ret';
+
+        # give a new file name
+        assert_screen 'yast2_ntp-client_save_log_as';
+        send_key 'alt-f';
+        type_string 'ntpclient.log';
```

Comment:
According to my experience, we may start typing here before we reach text field, so to mitigate these risks, I would put wait_still_screen in between. But as long as it worked before, should be fine.

Reply:
It's the same code as before, but I can add a `wait_still_screen` to be sure, yes.

Reply:
be careful with `wait_still_screen`. In most cases it might not seem like it but an `assert_screen` is faster and therefore more efficient because we can immediately check the screen for the expected screen. With `wait_still_screen` we invest *at least* one full second, in most cases more. Use `wait_still_screen(1)` for a start and increase the still time if 1 second is not enough.

Reply:
I could also let it as before, as it seems that it was working.

@rwx788, @okurz, what do you think is the best to do?

Reply:
I recommend one of the following, you can decide:
1. run at least 20 iterations of a scenario covering this locally to see if it makes a problem and add a synchronisation if you see a problem
* leave it as is

Reply:
As we have mentioned above, let's leave it as it is, as it worked before and if there is any issue, we can address it later

Reply:
Ok, I leave it as is.